<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>download</class>
 <widget class="QWidget" name="download">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>613</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Download</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>106</y>
     <width>601</width>
     <height>311</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>599</width>
      <height>309</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>55</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="mapping_grid">
      <item row="0" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Anki Field</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>Text</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tags</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>&lt;None&gt;</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Notion Column</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Anki Type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Placeholder</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Notion Type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>601</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="download_button">
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progress_bar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="progress_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>601</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0/0 Downloaded</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Notion Database</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="notion_database_select"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Sync Mode</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="sync_mode">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Append</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Soft Merge</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hard Merge</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Primary Key</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="primary_key"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Anki Deck</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="anki_deck_select"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
